Windows/MainWindowMenu.cpp

```diff
@@ -328,30 +386,30 @@ namespace MainWindow {
 		}
 	}
 
-	void SaveStateActionFinished(bool result, void *userdata) {
+	static void SaveStateActionFinished(bool result, void *userdata) {
 		PostMessage(MainWindow::GetHWND(), WM_USER_SAVESTATE_FINISH, 0, 0);
 	}
 
-
+	// not static
```

Comment:
Why?  Doesn't really look different from `setTexScalingType`, which is static?

-[Unknown]